packages/Webkul/Admin/src/Resources/lang/fa/app.php

```diff
@@ -1295,11 +1295,11 @@
             'locale-options' => 'گزینه های واحد',
             'weight-unit' => 'واحد وزن',
             'admin-page-limit' => 'موارد پیش فرض در هر صفحه (مدیر)',
-            'email-settings'    => 'Email Settings',
-            'email-sender-name' => 'Email Sender Name',
-            'shop-email-from'   => 'Shop Email Address [For sending emails]',
-            'admin-name'    => 'Admin Name',
-            'admin-email' => 'Admin Email',
+            'email-settings'    => 'إعدادات البريد الإلكتروني',
```

Comment:
@AayushBhatt-webkul
remove the persian locale changes as this issue is only reported for ar locale